lib/sendcloud/operations/tracking_request.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Sendcloud
+  module Operations
+    class TrackingRequest < Operation
+      TRACKING_PARAMS = [
+        :tracking_number,
+        :tracking_url,
+        :carrier
```

Comment:
We need `:status` here as well for the tracking status.

Reply:
Thanks @gdackys I'll add this.

Reply:
That's now added @gdackys